libkbfs/journal_block_server.go

```diff
@@ -16,14 +16,17 @@ var _ BlockServer = journalBlockServer{}
 
 func (j journalBlockServer) Get(
 	ctx context.Context, tlfID TlfID, id BlockID, context BlockContext) (
-	[]byte, BlockCryptKeyServerHalf, error) {
+	data []byte, serverHalf BlockCryptKeyServerHalf, err error) {
 	if tlfJournal, ok := j.jServer.getTLFJournal(tlfID); ok {
+		defer func() {
+			err = translateToBlockServerError(err)
+		}()
 		data, serverHalf, err := tlfJournal.getBlockDataWithContext(
 			id, context)
 		switch err.(type) {
 		case nil:
 			return data, serverHalf, nil
-		case BServerErrorBlockNonExistent:
```

Comment:
The bug that CI caught (thanks!) was that the journal was returning `blockNonExistentError` so this case wasn't being hit.